features/bevragen/persoon/uitsluiting-kiesrecht/dev/fields-gba.feature

```diff
@@ -0,0 +1,117 @@
+# language: nl
+
+@gba
+Functionaliteit: GBA Persoon: uitsluiting kiesrecht velden vragen met fields
+
+	Scenario: persoon heeft geen uitsluiting kiesrecht gegevens
+		Gegeven de persoon met burgerservicenummer '000000164' heeft de volgende gegevens
+    | geslachtsnaam (02.40) |
+    | Janssen               |
+		Als personen wordt gezocht met de volgende parameters
+    | naam                | waarde                          |
+    | type                | RaadpleegMetBurgerservicenummer |
+    | burgerservicenummer | 000000164                       |
+    | fields              | uitsluitingKiesrecht            |
+    Dan heeft de response een persoon zonder gegevens
+
+	Scenario: 'aanduiding uitgesloten kiesrecht (38.10)' heeft geen waarde
+		Gegeven de persoon met burgerservicenummer '000000164' heeft de volgende 'kiesrecht' gegevens
+    | einddatum uitsluiting Europees kiesrecht (31.30) |
+    | morgen                                           |
+		Als gba personen wordt gezocht met de volgende parameters
+    | naam                | waarde                          |
+    | type                | RaadpleegMetBurgerservicenummer |
+    | burgerservicenummer | 000000164                       |
+    | fields              | uitsluitingKiesrecht            |
+    Dan heeft de response een persoon zonder gegevens
+
```

Comment:
Tja...

Reply:
Gelukkig heeft Frans hierover uitsluitsel gegeven:
"Ik heb het nagekeken. Het komt niet voor dat de datum uitsluiting kiesrecht of de datum Europees kiesrecht gevuld is en de bijbehorende aanduiding niet.
En er zijn ook controles voor deze eisen in het Logisch Ontwerp. Als de datum wel aanwezig is en de aanduiding niet (of andersom) levert dat een bevinding op in de Kwaliteitsmonitor voor de betreffende gemeente. De gemeente moet die bevinding dan z.s.m. oplossen.
Het aantal inwoners dat uitsluiting kiesrecht heeft, is overigens erg klein (25) en daalt nog steeds. Dus de kans dat je er daar een van opvraagt is in de buurt van 1 op de miljoen."